cluster/src/main/java/com/hartwig/pipeline/execution/vm/BashStartupScript.java

```diff
@@ -1,39 +1,59 @@
 package com.hartwig.pipeline.execution.vm;
 
 import static java.lang.String.format;
+import static java.util.Arrays.asList;
 import static java.util.stream.Collectors.joining;
 
 import java.util.ArrayList;
 import java.util.List;
 
+import com.hartwig.pipeline.execution.vm.storage.StorageStrategy;
+
 public class BashStartupScript {
     static final String JOB_SUCCEEDED_FLAG = "JOB_SUCCESS";
     static final String JOB_FAILED_FLAG = "JOB_FAILURE";
-    private static final String LOG_FILE = VmDirectories.OUTPUT + "/run.log";
+    private static final String LOG_FILE = "/tmp/run.log";
```

Comment:
don't love this change, as its a little confusing when logging into the VM that log is in /tmp. I think it might be better to separate the things done early in the VM creation (which I guess is the reason for this) into a separate log file from the actual tool logs. Maybe have a startup.log and run.log.  I think the former will be checked far less than the latter.

Reply:
I can understand that; what if we put it in a more-standard place, like `/var/log`? That's what I'd really prefer to do, then just copy it at the end. I'm not sure why I didn't do that in fact.

You're right it's done so we have a single log for the whole run.